lib/fa-harness-tools/check_schedule.rb

```diff
@@ -22,14 +21,10 @@ def verify?
       @logger.info("operating in the #{@timezone} timezone")
       @logger.info("local time is #{@now}")
 
-      permitted = false
-      case @now.wday
-      when 1..4
-        @logger.info("deployments are allowed between 9am to 4pm today (Mon-Thu)")
-        permitted = true if @now.hour >= 9 && @now.hour < 16
-      when 5
-        @logger.info("deployments are allowed between 9am to 12pm today (Fri)")
-        permitted = true if @now.hour >= 9 && @now.hour < 12
+      permitted = @schedules.any? do |schedule|
+        can_run = schedule.can_run?(time: @now)
+        @logger.info("deployments are allowed due to the following schedule: #{schedule.to_s}") if can_run
```

Comment:
It's a pretty minor point, but should we also log the configuration in case it doesn't pass the check? Currently it'll just the log the time & timezone but not say what it's configured to allow.

Reply:
Sure!

Reply:
Instead of doing it here though, should I do it in the case it fails overall? I'm just wondering if it would be confusing to see 'schedule failed' logs in the accepted stream